lib/wallaroo/core/network/connections.pony

```diff
@@ -817,6 +818,159 @@ actor Connections is Cluster
     @printf[I32]("SLF: TODO Connections.update_worker_data_service: anything with _data_conns iteration & update?\n".cstring())
     // TODO ^^^^
 
+  be connector_stream_notify(leader: WorkerName, worker_name: WorkerName,
```

Comment:
These still need to be removed from `Connection` and added to a separate file on a primitive.  Notice that they don't use any of `Connection`'s fields except auth.  Each method on the primitive can take `Connections` as an argument if you want them to handle sending the control message.  But in any case, `Connections` handles the connections between workers, it shouldn't know anything about channel message specifics.

Reply:
@jtfmumm done, please mark as resolved if it matches your expectations